package.json

```diff
@@ -20,16 +20,17 @@
     "espree": "^3.5.2",
     "mkdirp": "^0.5.1",
     "parse5": "^2.2.2",
-    "polymer-analyzer": "^3.0.0-pre.2",
-    "source-map": "^0.5.6"
+    "polymer-analyzer": "^3.0.0-pre.7",
+    "source-map": "^0.5.6",
+    "vscode-uri": "^1.0.1"
   },
   "devDependencies": {
     "@types/chai": "^3.4.30",
     "@types/clone": "^0.1.30",
     "@types/mocha": "^2.2.29",
     "@types/node": "^6.0.33",
     "@types/parse5": "^2.2.33",
-    "@types/source-map": "^0.5.0",
+    "@types/source-map": "=0.5.2",
```

Comment:
Latest release of `@types/source-map` is missing type definitions.  Pinning this to 0.5.2 until they publish a fix.

Reply:
Verified this is still an issue.